Cannot find module '@bounce/iframe' or its corresponding type declarations.ts

**Describe the bug**
"Cannot find module '@bounce/iframe' or its corresponding type declarations.ts(2307)"

**To Reproduce**
Steps to reproduce the behavior:
Created a new expo app
installed library: npx expo install @bounceapp/iframe react-native-webview
then npm run web
I get that error even though the library is installed and I see it in the package.json

**Expected behavior**
To see a iframe

--- a/README.md
+++ b/README.md
@@ -27,7 +27,7 @@ npx expo install @bounceapp/iframe react-native-webview
 ```tsx
 import React from "react"
 
-import { Iframe } from "@bounce/iframe"
+import { Iframe } from "@bounceapp/iframe"
 
 const App = () => (
   <Iframe uri="https://usebounce.com/careers" style={{ flex: 1 }} />

--- a/src/Iframe.tsx
+++ b/src/Iframe.tsx
@@ -6,7 +6,7 @@ import { IframeProps } from "./types"
  * ```tsx
  * import React from "react"
  *
- * import { Iframe } from "@bounce/iframe"
+ * import { Iframe } from "@bounceapp/iframe"
  *
  * const App = () => (
  *   <Iframe uri="https://usebounce.com/careers" style={{ flex: 1 }} />

